### DISCUSSION TITLE
posts/2024/04/field-essay-52/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#50] [General]
# posts/2024/04/field-essay-52/

https://guozheng.rbind.io/posts/2024/04/field-essay-52/

**Comment by earfanfan** [comment]
额，其实我最近看你的博客总有一种疑惑，讲真，学习社会学、人类学的学生们是不是在参与田野调查的过程中极其、非常、迫切地需要一种使命感，或者说意义感，就是你要相信你做的事情是有意义的，有了这种信仰你才能继续做下去？

前面我看到你写“撬动社区的力量”时其实就没看明白，从我的视角看“撬动社区的力量”似乎是一个目标或者说一系列的行为，但是看不懂（或者只是你没写出来、或者是我还没看到）要撬动社区的力量去做什么呢？于是乎，我也没看明白，为什么“致力于某个社区的参与”一定要有什么影响，为撒一定要影响社区或者改变社区呢，这就是我看的时候感到迷惑的地方。

之前我读过的一些书，比如《江村经济》《小镇喧嚣》《中县干部》《以利为利》等书籍，作者都有提到参与过田野调查，但是没见到他们写想要对调查对象产生什么影响，而都是希望从调查对象那里获取些什么，因为他们都写成了书，显然他们得到了想要的东西。

**Reply by residualsun1** [reply]
> 额，其实我最近看你的博客总有一种疑惑，讲真，学习社会学、人类学的学生们是不是在参与田野调查的过程中极其、非常、迫切地需要一种使命感，或者说意义感，就是你要相信你做的事情是有意义的，有了这种信仰你才能继续做下去？

我觉得不是，至少就大多数人而言，或许都不会“**极其、非常、迫切地需要一种使命感，或者说意义感**”。其实看到你说我的博客给你这样的感觉，我想自己在某些地方的理解与表达上都偏颇了一些——这是不是我的一种自我催眠和陶醉？当听见别人发言，或者看见别人写作时，如果有什么总喜欢往宏大的价值和意义上靠，我自己也会觉得有点疑惑——真的是这样吗？现实生活中，真的有那么为爱发电吗？我自己就不大相信，因为转头就是现实的各种一地鸡毛、充斥的欲望和数不尽的怠惰。我想起马林诺夫斯基写的[《西太平洋上的航海者》](https://book.douban.com/subject/26732332/)和[《一本严格意义上的日记》](https://book.douban.com/subject/26022684/)，前者奠定了马氏在人类学界的地位，他在里面表现出了对土著人的赞美、肯定以及一系列积极的情感，但在他的日记（他死后，妻子便将这日记公之于众了）里，他将他的苦恼，对某些土著人的愤恨、轻蔑甚至是某些与性有关的想法都写在了里面，在人类学界引起了巨大的争议。

让你产生疑惑，我想有几个原因：1、文字的表述和现实的境况之间有着一定的距离。我的理解是从田野离开后，会想把田野中的美好尽量以较为充沛的情感表达和传达给别人，但我或许没把握好这个度，让你觉得似乎进入田野就必须带有各种崇高的情感。这也是我觉得师姐的手记让我感到折服的一个原因——她当然也有不少修饰的语言，但内容平实，流露的情绪如涓涓细流，却又很有力量。相比而言，她写的比我更加生活化，接近生活。2、我还是执着于师姐说的“人类学到头来终究是个自私的学科“。我们都进入了同样的田野，我害怕自己什么也没“看见”——这或许意味着我所说的”参与“、“改变”也可能是自己建构和创造出来的，迫切地想找到一个与她不同的视角。但这些也确实是我的经历，构成了我的理解，我仍然会选择相信这份参与。但我反对**强调**使命感与意义感，当反复强调价值，具体的行动可能就被消解了。毕竟有谁每天顶着一个崇高的信仰生活，能毫无压力顺利做事呢。饭团后来也和我说，回村以后做的事情都没想过有多大的目标和去改变什么，内心想着能影响一两人就好了。

我觉得你和我讲的那句话很好：**做具体的事情，感受具体的感受**。

想来博文中充斥着不少我的刻意。希望之后不会再给你留下”学习社会学、人类学的学生们”“在参与田野调查的过程中极其、非常、迫切地需要一种使命感，或者说意义感，要相信做的事情是有意义的，有了这种信仰才能继续做下去“的印象。

> 前面我看到你写“撬动社区的力量”时其实就没看明白，从我的视角看“撬动社区的力量”似乎是一个目标或者说一系列的行为，但是看不懂（或者只是你没写出来、或者是我还没看到）要撬动社区的力量去做什么呢？于是乎，我也没看明白，为什么“致力于某个社区的参与”一定要有什么影响，为撒一定要影响社区或者改变社区呢，这就是我看的时候感到迷惑的地方。

来到“撬动社区的力量”。我自己其实也没有明白它究竟是什么，朦朦胧胧的，或许就是我觉得我们和村民的联系越来越紧密，在脑子里时常会想象出一条纽带。这让我觉得我们可以发动他们一起做一些特殊的事情，比如邀请村民给小朋友讲故事，动员小朋友们的父母参与到小朋友的活动中。最后要做的事情是什么，我也没有答案，好像讲来讲去，都是希望能先让大家联系起来、熟悉起来，甚至亲近起来——我们更了解他们是谁，他们理解并支持我们在村子里救鲎、给小朋友们做各种活动、想要了解村庄的仪式等等。我觉得这还是一个过程，举办神明座谈会时，饭团也在思索如何和村民讲清楚我们这群人究竟在村子里干什么，和他们有啥关系。如果非要有一个答案，官方的就是项目书所写“可持续渔村发展模式探索”，但在具体的行动中，我觉得有很多丰富而具体的内容在里面。

至于影响和改变社区，我现在的理解是这个过程一直在进行，只是我在反复强调，或许因此显得突兀和刻意了。

> 之前我读过的一些书，比如《江村经济》《小镇喧嚣》《中县干部》《以利为利》等书籍，作者都有提到参与过田野调查，但是没见到他们写想要对调查对象产生什么影响，而都是希望从调查对象那里获取些什么，因为他们都写成了书，显然他们得到了想要的东西。

或许也有的作者以自己的方式做了回馈，突然想到厦门大学的彭兆荣教授就专门写了一本《[生命中的田野](https://book.douban.com/subject/36180137/)》，据说讲的是自己在田野里相遇的那些人，以及他们之间发生的故事，但我没看过。然后，在我的不成熟的认知和理解里，田野调查有很多种，但人类学的犹为特殊（尽管也有许多人强调人类学的田野调查不要对调查对象产生什么影响），我将它浪漫化了——例如我在凯博文（医学人类学家）的《道德的重量》中读到的艾迪的故事（艾迪不是人类学家）；台湾中研院的人类学家黄应贵协助布农人做经济建设，后来对自己帮倒忙的反思；在胡台丽拍摄的[《兰屿观点》](https://movie.douban.com/subject/2272268/)中，一个镜头内，有一个雅美族人说人类学者在兰屿做的研究越多，对雅美人的伤害就越深，把兰屿当做得到某一个社会地位的工具，并没有回馈给它所研究的对象。在我眼中，似乎只有人类学会关注这些议题，关于他者的故事，关于研究者对被研究者的态度、关系和情感，怀有感恩的情感，同时去追问——如何回馈。同时，值得称道的人类学家不会在写成了书，显然得到了自己想要的东西后便拍拍屁股离开，或许有的人确实没对田野中的人提供什么帮助，但结合我在村子里的生活，我的理解是，长期的田野调查会让我以同时是异乡人，又是试图作为这里一部分的当地人的身份感受到村民们的生活、情绪、善意，我不会完全是此间的人，但我却可以对他们产生亲近的情感，这会让我觉得想做一些什么。当感受到饭团所做的活动将岛屿上不同村子的村民聚集在一起讨论共同的历史和神明故事，一起带着渔民的后代参观海洋大学时，这些便构成了我自己对田野的理解：或许我们真的能对这些村民有某些影响，而这些影响现在看来或许微不足道，但我们相信对未来是好的。至于究竟如何，就交给时间吧。

你提的问题也是我自己要澄清的问题，但写完上面这些文字时，我觉得还是有些含糊。先写下来，待我后面想得清楚一些，还会回来修改的。

**Reply by earfanfan** [reply]
额，我犯了一个错，作为一个大外行弄混了社会学和人类学。前面提到的那几本书其实都算是社会学领域的，看完你写的，我在想是不是社会学强调的田野调查伦理和人类学的是不一样的。

那你慢慢想吧，来日方长，熬夜多了白天会没精神的说。

**Reply by residualsun1** [reply]
> 额，我犯了一个错，作为一个大外行弄混了社会学和人类学。前面提到的那几本书其实都算是社会学领域的，看完你写的，我在想是不是社会学强调的田野调查伦理和人类学的是不一样的。

不会的不会的，我认为这只是在我这里不一样，只是我把它浪漫化了，换句话说这些只是我的一面之词。说起“外行”，我只是一个正在学习的学生，说来我觉得自己远远没有入门，大多数时候在自说自话，所以我认为咱们之间的讨论没有对错之分，只有阅读和实践方向上的差异。我反而觉得你问的问题对我来说还很重要，一方面至少我不会被困在自己理解的世界，会知道别人咋想，然后再回来看看自己；另一方面，你的疑惑还会把我藏在心底里不敢面对的一些东西揪出来——是我没弄明白的模糊、有一些刻意的表达、一些强说新词。我很佩服很佩服师姐的一个原因，就是觉得她写的东西真实、平实又很有力量，不需要依附许多理论，也不需要攀上什么价值和意义，在平静的文字和娓娓道来的叙述里自然而然地表达情感。

人总会下意识地保护自己的，我想我之前的回复较真了一些，同时多多少少也想下意识地为自己的一些刻意表达做辩护。不知道会不会给你较真或者严肃啥的感觉，但其实我很高兴能接收到你真实的疑惑🫣，因为这些疑惑恰恰也是一些我在心底总不自觉想逃避，并时常用语言去为自己掩护的问题，比如，对我自己来说，我真的那么需要那么多意义吗，我必须去强调它们吗，那模糊的撬动力量究竟是啥……吭哧吭哧，希望我之后能更审慎、清晰地表达自己的想法🧐。

> 那你慢慢想吧，来日方长，熬夜多了白天会没精神的说。

积习难改இ௰இ，尽力而为。

**Reply by earfanfan** [reply]
我说我犯了一个错，确实就只是觉得自己弄混了社会学和人类学，今早遛狗的时候，我还问了一下豆包 AI，“人类学和社会学有什么区别？”，这是它给出的答案。

>一、研究对象</br>
人类学：主要研究人类的生物特征和文化特征，包括人类的体质、语言、文化、社会结构、历史发展等方面。</br>
社会学：主要研究社会现象、社会结构、社会行为、社会关系等方面。</br>
二、研究方法</br>
人类学：采用田野调查、民族志等方法，深入实地进行观察、访谈、参与等，以获取第一手资料。</br>
社会学：运用多种研究方法，如问卷调查、统计分析、实验等，对社会现象进行量化和质性研究。</br>
三、学科起源</br>
人类学：起源于对异文化的研究和探索。</br>
社会学：起源于对工业社会的研究和分析。</br>
四、社会功能</br>
人类学：帮助人们更好地理解人类的多样性和文化差异，促进跨文化交流和理解。</br>
社会学：为社会政策的制定和社会问题的解决提供理论支持和实证研究。</br>

黄同学呀，你可以淡定一点，我有单方面认为你是我的朋友，所以你较真或者严肃或者批判自己的文字，在我看来只是组成你的一部分，从未让我感到半分不适，在与我的交流上其实你的表达可以更放心一点。我发出疑惑只是发出疑惑，我的疑惑恰好引出你的困惑，这在我看来就是“哦，原来是这样啊”。

最后，哈哈，我也要刻意地表达，刻意地强调一点，我犯错其实是与你无关的，你可以不用有什么心理压力，不要想象成是你造成的，我跟你是独立的个体，都有各自的人生考验要去面对，你真得完全不用背负别人的一些什么。此时，我要对你说一句超老套的话：加油，年轻人，别害怕犯错哦。

**Reply by residualsun1** [reply]
> 黄同学呀，你可以淡定一点，我有单方面认为你是我的朋友，所以你较真或者严肃或者批判自己的文字，在我看来只是组成你的一部分，从未让我感到半分不适，在与我的交流上其实你的表达可以更放心一点。我发出疑惑只是发出疑惑，我的疑惑恰好引出你的困惑，这在我看来就是“哦，原来是这样啊”。

放心啦:D

> 最后，哈哈，我也要刻意地表达，刻意地强调一点，我犯错其实是与你无关的，你可以不用有什么心理压力，不要想象成是你造成的，我跟你是独立的个体，都有各自的人生考验要去面对，你真得完全不用背负别人的一些什么。

不会的，因为一开始也没有认为你犯错，只是有点担心你会真的以为学我们这个专业的学生会是这样，而我自认还没入门。不过就算是犯错了，现在和之后我也不会背负啥心理压力的。

> 此时，我要对你说一句超老套的话：加油，年轻人，别害怕犯错哦。

嗯 : )，虽然转头可能还会继续担心，但或许正因如此，这句话才总有箴言的意味吧。
